src/index/blockfilterindex.h

```diff
@@ -12,6 +12,8 @@
 #include <index/base.h>
 #include <util/hasher.h>
 
+#include <string>
```

Comment:
Isn't it the case that we don't need the string include here, or in `txindex.h` or `coinstatsindex.h`?

Reply:
Okay